pkgs/by-name/fi/fider/server.nix

```diff
@@ -0,0 +1,30 @@
+{
+  buildGoModule,
+
+  pname,
+  version,
+  src,
+  vendorHash,
+}:
+
+buildGoModule {
+  inherit version src vendorHash;
+  pname = "${pname}-server";
+
+  patches = [
+    ./0001-disable-etc-copy.patch
```

Comment:
Maybe you should add a comment explaining why the patch is needed, plus a link to the issue.

Reply:
Actually let me have another look at the copy mechanism, I went through the code again just now and wonder if there is a better solution than disabling it...

Reply:
OK I’ll follow it.